platform/api.yml

```diff
@@ -366,10 +366,10 @@ paths:
     $ref: "./paths/containers/instances/tasks.yml"
   "/v1/containers/{containerId}/instances/{instanceId}/volumes":
     $ref: "./paths/containers/instances/volumes.yml"
-  ? "/v1/containers/{containerId}/instances/{instanceId}/telemetry/resources/report"
```

Comment:
what formatter do you use? this keeps getting overridden and my redhat yaml plugin formats it with these so we keep fucking up these lines.